internal/pkg/service/stream/definition/repository/sink/sink_enable_test.go

```diff
@@ -92,6 +92,27 @@ func TestSinkRepository_Enable(t *testing.T) {
 		require.NoError(t, err)
 		assert.True(t, sink.IsEnabled())
 		assert.True(t, sink.IsEnabledAt(now))
+		assert.Equal(t, definition.VersionNumber(3), sink.VersionNumber())
+	}
+
+	// Rollback to the disabled state
+	// -----------------------------------------------------------------------------------------------------------------
+	{
+		now = now.Add(time.Hour)
+		sink, err := repo.RollbackVersion(sinkKey, now, by, 2).Do(ctx).ResultOrErr()
```

Comment:
i see that we rollback within the test to state 2 which I supposed is Create - OK, Get -OK and Disable - OK <- here.
Why the definition.VersionNumber is 4? It is just autoincrement of versions? I see 3 version or 5 but not 4. Could you explain why the VersionNumber is 4? Is it 3 versions + 1 rollback?

Reply:
[tests: Add more version asserts](https://github.com/keboola/keboola-as-code/pull/1789/commits/8af8087e1bd94548fe22e0c30ca38661ce6b97d7)

Reply:
Thank you, so read operations are not counted as versionNumber obviously. Rollback is to exact version.